src/styles/blocks/card.scss

```diff
@@ -0,0 +1,116 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    align-self: center;
+
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin: 0;
+    padding-top: 4px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text-color;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    padding-top: 16px;
+    width: 166px;
+    height: 16px;
+  }
+
+  &__count-review {
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    width: 53px;
+    height: 16px;
+    color: $main-text-color;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    width: 166px;
+    height: 18px;
+    padding-top: 24px;
+
+    &-label {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-text-color;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      text-align: right;
+      color: $main-text-color;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 166px;
+    height: 40px;
+    margin-top: 16px;
+    margin-bottom: 15px;
```

Comment:
There is an inconsistency in the margin values. The `margin-top` is set to 16px, while the `margin-bottom` is set to 15px. If this is intentional, it's fine, but if not, consider making them consistent.